CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+cmake_minimum_required(VERSION 3.10.2 FATAL_ERROR)
+
+#============================================================================
+# Initialize the project
+#============================================================================
+project(ignition-usd1 VERSION 1.0.0)
```

Comment:
You could keep it in major version zero until it's stable. We've done this with core Gazebo libraries in the past, Acropolis shipped with `ign-launch0`, so we could break API / ABI within Acropolis

Reply:
https://github.com/gazebosim/gz-usd/pull/1/commits/a21d5b60525204df1c8157491b0578ac41289c7c